First shot on integration tests for copr build (#199)
packit-service/packit-service: CONTRIBUTING.md

make check_in_container
```

## Openshift tests using requre
This testsuite uses [requre project](https://github.com/packit-service/requre) project to
to store and replay data for tests.

### General requirements
 * Set up docker and allow your user access it:
   ```bash
   sudo dnf -y install docker
   sudo groupadd docker
   sudo usermod -a -G docker $(whoami)
   echo '{ "insecure-registries": ["172.30.0.0/16"] }' | sudo tee  /etc/docker/daemon.json
   sudo systemctl restart docker

   newgrp docker
   ```
 * Install and run local openshift cluster:
   ```bash
   sudo dnf install origins-client python3-openshift
   oc cluster up --base-dir=/tmp/openshift_cluster
   ```

### Data regeneration

>  * copy secrets direcotory to top directory of packit-service

Reviewer: ```suggestion
 * copy secrets directory to the root of this project: `cp -ar path/to/secrets/ secrets/
```

>  * remove all necessary files what you want to regenerate

Reviewer: ```suggestion
 * remove files which you want to regenerate:
```

   ```bash
   rm -r tests_requre/test_data/test_*
   ```

>  * Run the regeneration (part of this step is also doing containers for ``worker`` and ``test_image``)

Reviewer: ```suggestion
 * Run the tests with the secrets - the response files will be regenerated (container images for `worker` and `test_image` are done in this step)
```
